wipasswd.swift

```diff
@@ -16,33 +17,40 @@ let kSecMatchLimitOneValue = kSecMatchLimitOne as NSString
 let kAirPortService = "AirPort"
 
 func getCurrentSsid() -> String? {
-    return CWWiFiClient.sharedWiFiClient().interface().ssid()
+    if #available(OSX 10.10, *) {
```

Comment:
Using -target option of swiftc command might be a better solution since without CWWiFiClient wipasswd is mostly useless. I'd change Makefile to do it.